src/NuGet.Clients/VsConsole/PowerShellHost/NuGetConsole.Host.PowerShell.csproj

```diff
@@ -105,6 +105,10 @@
       <Project>{306cddfa-ff0b-4299-930c-9ec6c9308160}</Project>
       <Name>NuGet.PackageManagement.VisualStudio</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\VisualStudio.Facade\NuGet.VisualStudio.Facade.csproj">
```

Comment:
why do we need to add this ProjectReference? I don't see any other change in PowerShellHost project...